packages/apollo-cli/src/commands/codegen/generate.ts

```diff
@@ -122,7 +126,9 @@ export default class Generate extends Command {
       {
         title: "Scanning for GraphQL queries",
         task: async (ctx, task) => {
-          const paths = await globby(flags.queries ? flags.queries.split('\n') : []);
+          const paths = withGlobalFS(() => {
+            return (flags.queries ? flags.queries.split('\n') : []).flatMap(p => fg.sync(p));
```

Comment:
`globby` had some weird incompatibilities with the in-memory volume, so I switched to `glob` here.